cmd/commands/subvolume.go

```diff
@@ -40,17 +40,13 @@ var listCmd = &cobra.Command{
 }
 
 var deleteCmd = &cobra.Command{
-	Use:                "delete",
-	Short:              "Deletes a stale subvolume.",
-	DisableFlagParsing: true,
-	Args:               cobra.ExactArgs(3),
-	Example:            "kubectl rook-ceph delete <subvolumes> <filesystem> <subvolumegroup>",
+	Use:     "delete",
+	Short:   "Deletes a stale subvolume.",
+	Args:    cobra.MinimumNArgs(2),
+	Example: "kubectl rook-ceph delete <filesystem> <subvolume> <subvolumegroup>",
```

Comment:
Let's use square brackets around optional args
```suggestion
	Example: "kubectl rook-ceph delete <filesystem> <subvolume> [subvolumegroup]",
```